scss/_variables.scss

```diff
@@ -1096,12 +1096,12 @@ $input-padding-y-lg:                    $input-btn-padding-y-lg !default;
 $input-padding-x-lg:                    $input-btn-padding-x-lg !default;
 $input-font-size-lg:                    $input-btn-font-size-lg !default;
 
-$input-bg:                              var(--#{$prefix}body-bg) !default;
+$input-bg:                              $white !default; // Boosted mod: instead of `var(--#{$prefix}body-bg)`
 $input-disabled-color:                  $gray-700 !default; // Boosted mod
 $input-disabled-bg:                     var(--#{$prefix}secondary-bg) !default;
 $input-disabled-border-color:           null !default;
 
-$input-color:                           var(--#{$prefix}body-color) !default;
+$input-color:                           $black !default; // Boosted mod: instead of `var(--#{$prefix}body-color)`
```

Comment:
I think this should remain `var(--#{$prefix}body-color)` but the problematic input (in the orange navbar) should have a black background (like https://system.design.orange.com/0c1af118d/p/42765f-global-search/b/2696bb).

Reply:
Not in regular navbars in the doc: https://deploy-preview-2030--boosted.netlify.app/docs/5.3/components/navbar/#supported-content